src/inc/corinfo.h

```diff
@@ -1312,6 +1312,7 @@ enum CORINFO_RUNTIME_LOOKUP_KIND
 struct CORINFO_LOOKUP_KIND
 {
     bool                        needsRuntimeLookup;
+    WORD                        runtimeLookupFlags;
```

Comment:
The range is 1-8 today. It should not be zero if needsRuntimeLookup is true (i think some other place in the codebase will catch that and assert on debug builds, and crossgen will not crash, but not compile the affected method)

---
In reply to: [62738274](https://github.com/dotnet/coreclr/pull/4519#discussion_r62738274) [](ancestors = 62738274)